public/javascripts/SVLabel/src/SVLabel/task/TaskContainer.js

```diff
@@ -467,53 +444,34 @@ function TaskContainer (navigationModel, neighborhoodModel, streetViewService, s
         var neighborhood = neighborhoodModel.currentNeighborhood();
         var currentNeighborhoodId = neighborhood.getProperty("regionId");
 
-        // If the neighborhood is 100% complete (across all users)
-        if (findNeighborhoodCompleteAcrossAllUsers(currentNeighborhoodId, finishedTask)) {
```

Comment:
Since you aren't using `findNeighborhoodCompleteAcrossAllUsers()` anymore, then the neighborhoodCompletion overlay won't be shown either since it was invoked within this function. Do we not want to show the neighborhood completed overlay if the user has finished the neighborhood (across all users)?

Reply:
Oh wow, I didn't expect `findNeighborhoodCompleteAcrossAllUsers` to also have the side effect of initiating the popup, so I assumed it was being checked elsewhere... That should really be separated into two functions. I'll do that quick and add the check back in.

Reply:
Okay, updated.

Reply:
Looks good!